addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -259,25 +259,27 @@
 					</control>
 					<control type="label">
 						<left>50</left>
-						<top>0</top>
+						<top>5</top>
 						<right>80</right>
-						<height>60</height>
-						<font>font14</font>
+						<height>30</height>
 						<aligny>center</aligny>
-						<label>$INFO[ListItem.Property(stream.codec)]$INFO[ListItem.Property(stream.resolution),$COMMA ]$INFO[ListItem.Property(stream.bitrate),$COMMA , kbps]$INFO[ListItem.Property(stream.fps),$COMMA , fps]$VAR[VideoStreamDialogVideoItemLabelVar, - [LIGHT],[/LIGHT]]</label>
+						<scroll>true</scroll>
+						<label>$INFO[ListItem.Property(stream.codec)]$INFO[ListItem.Property(stream.hdrtype, - )]$INFO[ListItem.Property(stream.resolution), - ]$INFO[ListItem.Property(stream.fps), - , fps]$VAR[VideoStreamDialogItemLabelVar, - [LIGHT],[/LIGHT]]</label>
```

Comment:
please restore the bitrate string where it was, without it its impossible to distinguish video tracks
![immagine](https://github.com/user-attachments/assets/2bc9000d-04d5-42fd-833c-fd17fbe15738)

Reply:
I removed it because everything I had displayed 0 kbps, and I didn't think you would have a choice where bitrate would be only difference. As you've shown this to be wrong I will look to restore bitrate, but see if I can do a check for a 0 value.

Reply:
oh right i had completely forgotten about the 0 yes a solution need to be found,
if not from skin side, from code by change the bitrate type from int to string value, and so when 0 return empty string

Reply:
New variable added so a conditional check for 0 can be done.
```
	<variable name="StreamBitrateVar">
		<value condition="!String.IsEqual(ListItem.Property(stream.bitrate),0)">$INFO[ListItem.Property(stream.bitrate), - , kbps]</value>
	</variable>
```